node/clients/oms.ts

```diff
@@ -44,10 +35,9 @@ export class OMSCustom extends OMS {
     })
   }
 
-  public listOrdersWithParams(params?: OrderListParams) {
-    return this.http.get<OrderList>(routes.orders, {
+  public listOrdersWithParams(params?: string) {
+    return this.http.get<OrderList>(`${routes.orders}?${params}`, {
```

Comment:
Any reason to move it from the object to a string? I feel that updating the param as string more difficult than just passing the object to the get request.

Reply:
It was not working when I encoded the email param. And I didn't find a solution, if you have time we can try together with the params object.